packages/rich-text-links/src/index.ts

```diff
@@ -12,15 +12,15 @@ export type EntityLinkNodeData = {
 /**
  * Given a rich text document, returns all entity links.
  */
-function getRichTextEntityLinks(document: Document): EntityLinks {
+function getRichTextEntityLinks(document: Document, type?: string): EntityLinks {
```

Comment:
May I ask you to annotate a new argument

Reply:
What do you mean by "annotate a new argument"? Can you show me some example?

Reply:
Sorry for the confusion, I meant something like this:
```js
/**
*  Extracts entity links from a Rich Text document.
*/
function getRichTextEntityLinks(
/**
*  An instance of a Rich Text Document.
*/
document: Document, 
/**
* Node type. Will be returned entity links only with given type.
*/
type?: string): EntityLinks
```

Reply:
I thought TypeScript annotations 😅  Updated.